elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/dimension/EntityDimension.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.dimension;
+
+import com.yahoo.elide.core.EntityDictionary;
+import com.yahoo.elide.datastores.aggregation.annotation.Cardinality;
+import com.yahoo.elide.datastores.aggregation.annotation.DimensionSize;
+
+/**
+ * Default implementation of {@link Dimension} that represents an entity as a dimension.
+ * <p>
+ * If the entity does not have {@link Cardinality} annotation on it, {@link EntityDimension} assigns a
+ * {@link #getDefaultCardinality() default cardinality}.
+ * <p>
+ * {@link EntityDimension} is thread-safe and can be accessed by multiple threads.
+ */
+public class EntityDimension extends AbstractDimension {
```

Comment:
Do we need this class?  Instead of having an interface, an abstract class, and a concrete class, can we collapse this to an interface and a ConcreteClass?